src/Nethermind/Nethermind.Mev/Execution/TxBundleExecutor.cs

```diff
@@ -57,17 +65,21 @@ private Block BuildBlock(MevBundle bundle, BlockHeader parent, UInt256? timestam
                 Beneficiary, 
                 parent.Difficulty,  
                 parent.Number + 1, 
-                parent.GasLimit, 
+                GetGasLimit(parent), 
                 timestamp ?? parent.Timestamp, 
                 Bytes.Empty)
             {
                 TotalDifficulty = parent.TotalDifficulty + parent.Difficulty
             };
 
+            header.BaseFeePerGas = BaseFeeCalculator.Calculate(parent, _specProvider.GetSpec(header.Number));
+
             return new Block(header, bundle.Transactions, Array.Empty<BlockHeader>());
         }
 
-        protected virtual Address Beneficiary => Address.Zero;
+        protected virtual long GetGasLimit(BlockHeader parent) => parent.GasLimit;
```

Comment:
Not sure about 1559 gas limit bumping on the fork block. Is this work for the 1559 fork block? 2*parent.GasLimit?

Reply:
This is for RPC, can't we just ignore it? Or we  can do similar thing we do in simulator and just use proper gas calculator? What do we do in eth_call?